docs/spec_checklist.md

```diff
@@ -44,3 +44,14 @@ reviews:
       1. Then links to the corresponding section of the spec.
       1. Then uses the same example as the spec but via pretty syntax which can
          be obtaining by running `stablehlo-opt`.
+  1. Check that the files related to implementing verification and type
+     inference constraints follow the guidelines as mentioned below:
+      1. Follow [guideline#1](https://github.com/openxla/stablehlo/blob/main/docs/reference_checklist.md#after-implementing-the-op)
```

Comment:
Typo: "guideline #1". (Missing whitespace - here and below).

Reply:
done.